_docs/it/schema.md

```diff
@@ -1,3 +1,794 @@
+persone come ad esempio "Linus Torvalds and all Linux contributors".
+
+Nel caso in cui non sia possibile nominare il maggior proprietario di
+copyright, è possibile omettere questa chiave; in questi casi, se il repository
+ha un file contenente il nome degli autori, è possibile puntare a quel file
+attraverso `legal/authorsFile` (vedi più sotto). 
+
+### Chiave `legal/repoOwner`
+
+* Tipo: stringa
+* Presenza: opzionale
+* Esempio: `"City of Amsterdam"`
+
+Questa stringa descrive l'entità che possiede il repository; questa essere
+o non essere la stessa che possiede il copyright del codice stesso. Ad esempio,
+nel caso di un fork del software originale, il `repoOwner` è probabilmente
+diverso dal `mainCopyrightOwner`.
+
+### Chiave `legal/authorsFile`
+
+* Tipo: stringa (percorso al file)
+* Presenza: opzionale
+* Esempio: `"doc/AUTHORS.txt"`
+
+Qualche software open-source adotta una convenzione che identifica il
+proprietario del copyright attraverso un file elencante tutte le entità che
+possiedono il copyright. Questo è comune nei progetti fortemente sostenuti
+dalla community ove esistono diversi contributori esterni e non c'è un chiaro
+singolo possessore del copyright. In questi casi, questa chiave può essere
+usata per riferirsi al suddetto file degli autori, usando un percorso relativo alla
+radice (root) del repository. 
+
+
+## Section `maintenance`
+
+Questa sezione fornisce informazioni sullo stato di manutenzione del software,
+utile per valutare se il software è attivamente sviluppato o meno. 
+
+### Chiave `maintenance/type`
+
+* Tipo: enumerate
+* Presenza: obbligatoria
+* Valori: `"internal"`, `"contract"`, `"community"`, `"none"`
+
+Questa chiave descrive come il software è attualmente manutenuto. 
+* `"internal"` significa che il software è manutenuto internamente dal
+  proprietario del repository; 
+* `"contract"` significa che c'è un contratto commerciale che lega un'entità
+  alla manutenzione del software;
+* `"community"` implica che il software è attualmente manutenuto da una o più
+  persone che offrono il loro tempo al progetto;
+* `"none"` significa che il software non è al momento manutenuto.
+
+### Chiave `maintenance/contractors`
+
+* Tipo: array di Contractor (vedi sotto)
+* Presenza: obbligatoria (se `maintainance/type` è `contract`)
+
+Questa chiave descrive l'entità o le entità, se ce ne sono, che attualmente
+hanno un contratto di manutenzione del software. Queste possono essere
+compagnie, organizzazioni o altri nomi collettivi. 
+
+### Chiave `maintenance/contacts`
+
+* Tipo: Lista di Contatti (vedi sotto)
+* Presenza: obbligatoria
+
+Uno o più contatti a chi sta mantenendo il software. 
+
+Questa chiave descrive le persone tecniche che attualmente sono responsabili
+della manutenzione del software. Tutti i contatti devono essere di una persona
+fisica, non una compagnia o un'organizzazione. Se qualcuno funge da
+rappresentante di un'istituzione, deve essere elencato tra `affiliation` dei
```

Comment:
La frase "deve essere elencato tra affiliation dei contatti" non mi è chiara

Reply:
Mi sono dimenticato un pezzo. In inglese è: `if somebody is acting as a representative of an institution, it must be listed within the affiliation of the contact.`
Direi meglio "Se un contatto funge da rappresentate di un'istituzione, questo rapporto deve essere esplicitato attraverso la chiave `affiliation`."